### ISSUE TITLE
New license request: HTMLTIDY

### ISSUE THREAD

**Issue by tisoft** [spdx/license-list-XML#1064]
_edited by @swinslow 2020-11-19_ -- note in comments below that the final accepted identifier was HTMLTIDY, not JTIDY

**1.** License Name: Java HTML Tidy - JTidy
**2.** Short identifier: JTIDY
**3.** License Author or steward: 
**4.** Comments: Prepared by spdx-license-diff 0.5.3
Used by JTidy, couldn't find a matching license in the list.
**5.** Standard License Header: 
**6.** URL: http://jtidy.sourceforge.net/license.html
**7.** OSI Status: Not Submitted

**Comment by johnmhoran** [comment]
@swinslow -- I'll tackle this one as well.

**Comment by johnmhoran** [comment]
From the research I did back in September:

tldr: The JTidy project is dated, not widely used and not maintained; but there is a GH repo looking for help, and there have been citations to the license in a handful of projects/products in the past few years.

No license author/steward listed.

Similar to zlib license: compare http://jtidy.sourceforge.net/license.html with https://spdx.org/licenses/Zlib.html 

Per SourceForge pages:

- Last updated 2012-10-09 (https://sourceforge.net/projects/jtidy/) 
- SourceForge link on http://jtidy.sourceforge.net/source-repository.html to "the online source repository" leads to a "not found on this server" page.  (http://jtidy.svn.sourceforge.net/viewvc/jtidy/) 

There's a GH repo: https://github.com/jtidy/jtidy 

- "The new home of JTidy updated to work with HTML5"
- Includes a link to one of the SourceForge pages.
- Includes a copy of the license (https://github.com/jtidy/jtidy/blob/master/LICENSE.txt) (2004-01-05 is most recent commit)
- README notes that "JTidy project on SourceForge.net seemed to fall into disrepair years ago and had not been updated for years.  . . .  [¶]  The project is looking for new contributors and project maintainers as it is currently unmaintained."
- Latest release: 2019-12-15 (https://github.com/jtidy/jtidy/releases/tag/1.0.2) 

Examples of usage in the wild:

Cisco 2018 (https://www.cisco.com/c/dam/en_us/about/doing_business/open_source/docs/JabberforiPhoneandiPad125v10.pdf) 
- "Open Source Used In Jabber for iPhone and iPad 12.5", pp. 48 et seq. including copy of license text titled "HTML Tidy"

qm-parcel 1.1.0
Copyright 2020 Cloudera Inc.  All Rights Reserved.
https://docs.cloudera.com/documentation/other/shared/licensefiles/cdppc7210/qm-parcel-1-1-0-notice.txt 
- component named "JTidy 4aug2000r7-dev"

Veritas Resiliency Platform 3.3 Third-Party Software License Agreements (2018-09-03)
https://sort.veritas.com/DocPortal/pdf/133634224-133634229-1 
- includes license notice and text for "JTidy r938", a "Third-Party Program[] . . . licensed under JTidy License."

**Comment by swinslow** [comment]
Thanks @johnmhoran, this is very helpful. The reference to "HTML Tidy" led me to https://github.com/htacg/tidy-html5/blob/next/README/LICENSE.md, which appears to be sizable (and notable enough to have [its own Wikipedia page](https://en.wikipedia.org/wiki/HTML_Tidy), anyway).

Given all of the above, and given that it appears to fall within the OSI definition from my read (looks to be [Zlib](https://spdx.org/licenses/Zlib.html)-ish), I'd be in favor of adding it. We can discuss during today's call.

**Comment by jlovejoy** [comment]
+1 to add. it's like zlib, but has a more robust disclaimer of warranties and extra bit at end.

I would consider the license text to be only from after "COPYRIGHT NOTICE:" (or make stuff above it optional?)

**Comment by swinslow** [comment]
Title: Java HTML Tidy License

ID: JTIDY

**Comment by swinslow** [comment]
no version number needed here

**Comment by johnmhoran** [comment]
The jtidy sourceforge project does indeed have a presence on the Wayback, with its license found here: https://web.archive.org/web/20180427223718/http://jtidy.sourceforge.net/license.html

**Comment by tisoft** [comment]
If you consider only the text after COPYRIGHT NOTICE as license, it would probably be better to use "HTML Tidy License" as license name with identifier "HTMLTIDY". HTML Tidy is the original project, JTidy is a java fork of that. The license text is basically the same, the JTidy license misses a single "held", added numbers for the restrictions and changed the spelling of acknowledg(e)ment: https://editor.mergely.com/OysSHMBn/

So in my opinion, they are the same license and I think the original one should be taken.

(I originally filed it for JTidy, since that came up in a license report of our product, I compared it with the existing SPDX licenses, but failed to search for the true origin)

**Comment by johnmhoran** [comment]
Thanks @tisoft -- that's my inclination as well.  @swinslow @jlovejoy -- what are your thoughts?

**Comment by swinslow** [comment]
+1 -- I don't know the history here but I defer to @tisoft, if HTML Tidy was the original project then makes sense to reference it in the name and identifier. Thanks!

**Comment by jlovejoy** [comment]
+1 on name and identifier comments made by @tisoft 

"The license text is basically the same, the JTidy license misses a single "held", added numbers for the restrictions and changed the spelling of acknowledg(e)ment: https://editor.mergely.com/OysSHMBn/"
We might want to have a closer look at this - is this something markup could accomodate?

**Comment by johnmhoran** [comment]
@swinslow  @jlovejoy  Created `.xml` and `.txt` files, committed and pushed (forgot to add this issue # to my commit message -- apologies!), opened a PR about 30 minutes ago, still waiting for the 2 Travis build jobs to start: https://github.com/spdx/license-list-XML/pull/1143

**Comment by swinslow** [comment]
#1143 is now merged, thank you @johnmhoran!
